pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactory.java

```diff
@@ -567,6 +567,10 @@ private void parseSingleRuleNode(RuleSetReferenceId ruleSetReferenceId, RuleSetB
             rule.setUsesTypeResolution();
         }
 
+        if (hasAttributeSetTrue(ruleElement, "metrics")) {
```

Comment:
Yes, that's one place, were the definition can be downloaded. But the master source is really in the main repo (under pmd-core): https://github.com/pmd/pmd/tree/master/pmd-core/src/main/resources

I'm not sure about whether we could just change the definition and keep the version/namespace the same (given that it is compatible) or whether we should change the namespace...